bin/citgm-all.js

```diff
@@ -97,6 +119,10 @@ function runCitgm (mod, name, next) {
   const runner = citgm.Tester(name, options);
   let bailed = false;
 
+  if (checkTags(app, mod, name)) {
+    return next(); // Skip this module
```

Comment:
Are we generally putting the `return` and the call to `next` on the same line? Technically we don't want to return the result of the callback... but I'm not sure that it matters. Good to be explicit + consistent though. Maybe time for a new lint rule?